packages/content/src/enrichments.ts

```diff
@@ -509,22 +561,20 @@ export const enrichContent = (
   content: IHubContent,
   requestOptions?: IEnrichContentOptions
 ) => {
-  // get enriched portal urls
-  const portalUrls = getPortalUrls(content, requestOptions);
+  // ensure defaults and enrich content with portal urls
+  const defaults: Partial<IHubContent> = {
+    errors: [],
+  };
+  const contentWithPortalUrls = {
+    ...defaults,
+    ...content,
+    ...getPortalUrls(content, requestOptions),
+  };
   // fetch any missing or requested enrichments
-  return fetchEnrichments(content, requestOptions).then(
-    (enrichments: Partial<IHubContent>) => {
-      const serverErrors = content.errors || [];
-      // merge derived portal URLs & fetched enrichments into content
-      const merged = {
-        ...content,
-        ...portalUrls,
-        ...enrichments,
-        // include any previous errors (if any)
-        errors: [...serverErrors, ...enrichments.errors],
-      };
+  return fetchEnrichments(contentWithPortalUrls, requestOptions).then(
+    (fetched) => {
       // enrich dates now that we have metadata (if any)
-      const enriched = _enrichDates(merged);
+      const enriched = _enrichDates(fetched);
```

Comment:
Since this is done for all content, it could be included in the pipeline...

Reply:
Odd - I can't leave a comment on a line that's not changed in this PR... but I'd also suggest looking at refactoring `getLayerContent` so it's also in the pipeline. As it stands, that fn does a check and early exits... so I think you could do a small refactor to make it part of the pipeline, which makes it all consistent.

Reply:
As the 😈 's advocate - `searchContent()` is async already, and wrapping a sync fn into a `Promise.resolve()` adds ~1 tick... now if you got 10,000 things back from the api, this may add a measurable amount of time, but it would still be minuscule compared to the first xhr to get the data from the api :)

Anyhow - I'm fine if you rename the fn, and leave it sync... I just see value in things being consistent w/in a sub system.